pkg/controller/controller_instance.go

```diff
@@ -354,22 +358,21 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) error {
 
 	glog.V(4).Infof("Processing Instance %v/%v", instance.Namespace, instance.Name)
 
-	// if the instance is marked for deletion, handle that first.
+	if instance.Status.AsyncOpInProgress {
```

Comment:
if you move this up to L328, we can remove a level of nesting in the checksum check